quodlibet/ext/events/autorating.py

```diff
@@ -15,7 +15,8 @@ class AutoRating(EventPlugin):
     PLUGIN_NAME = _("Automatic Rating")
     PLUGIN_DESC = _("Rates songs automatically when they are played or "
                     "skipped. This uses the 'accelerated' algorithm from "
-                    "vux by Brian Nelson.")
+                    "vux (Vacillating Utilitarian eXtemporizer) "
```

Comment:
FYI you can now use markup, if you wanted to include a hyperlink instead / as well, via `PLUGIN_DESC_MARKUP`